components/org.wso2.carbon.identity.recovery/src/main/java/org/wso2/carbon/identity/recovery/handler/ConfigStoreFunctionalityLockPropertyHandler.java

```diff
@@ -68,7 +69,8 @@ public static ConfigStoreFunctionalityLockPropertyHandler getInstance() {
                                             functionalityIdentifier);
                     properties = new ResourceToProperties().apply(resource);
                 } else {
-                    throw new UnsupportedOperationException("User Functionality properties are not configured.");
+                    log.trace("User Functionality properties are not configured. Resorting to default values.");
```

Comment:
Any reason to add trace logs here rather than debug logs?

Reply:
Resolved with https://github.com/wso2-extensions/identity-governance/pull/455/commits/fa99fda5efb1f0857820f29f792c243b024ef32a.